src/main/java/com/minecolonies/coremod/entity/pathfinding/PathJobFindWater.java

```diff
@@ -51,7 +51,7 @@
         /**
          * If the pond is empty.
          */
-        public boolean  isEmpty;
+        public boolean isEmpty;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make isEmpty a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)